<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>663</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>17</green>
        <blue>17</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>180</red>
        <green>180</green>
        <blue>180</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>17</green>
        <blue>17</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>198</red>
        <green>198</green>
        <blue>198</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>17</green>
        <blue>17</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>180</red>
        <green>180</green>
        <blue>180</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>17</green>
        <blue>17</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>198</red>
        <green>198</green>
        <blue>198</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>54</red>
        <green>54</green>
        <blue>54</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>42</red>
        <green>42</green>
        <blue>42</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>17</green>
        <blue>17</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>22</red>
        <green>22</green>
        <blue>22</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>180</red>
        <green>180</green>
        <blue>180</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>200</red>
        <green>200</green>
        <blue>200</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>35</red>
        <green>35</green>
        <blue>35</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>34</red>
        <green>34</green>
        <blue>34</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: consolas;
	color: rgb(200, 200, 200);
	background-color: rgb(35, 35, 35);
}

/* QLabel */
QLabel.standard {
	font-size: 10pt;
	background-color: transparent;
}

/* Menubar */
QMenuBar {
	background: rgb(50, 50, 50);
}

QMenuBar::item  {
    background-color: transparent;
	color: rgb(200, 200, 200);
}

QMenuBar::item:selected {
    background-color: rgb(100, 100, 100);
}

QMenuBar::item:pressed {
    background-color: rgb(200, 200, 200);
	color: rgb(20, 20, 20);
}

QMenuBar::item:disabled {
    background-color:transparent;
	color: rgb(100, 100, 100);
}

/* QMenu */
QMenu {
	background: rgb(50, 50, 50);
}

QMenu::separator {
	height: 1px;
	background-color: rgb(100, 100, 100);
	margin: 5px, 5px, 5px, 5px;
}

QMenu:disabled {
	color: rgb(100, 100, 100);
}

/*QMenu::indicator:non-exclusive:unchecked {
	image: url(:/Resources/Images/checkbox_off.png);
}

QMenu::indicator:non-exclusive:unchecked:selected {
	image: url(:/Resources/Images/checkbox_off.png);
}

QMenu::indicator:non-exclusive:checked {
	image: url(:/Resources/Images/checkbox_on.png);
}

QMenu::indicator:non-exclusive:checked:selected {
	image: url(:/Resources/Images/checkbox_on.png);
}

QMenu::indicator:exclusive:unchecked {
	image: url(:/Resources/Images/radiobutton_off.png);
}

QMenu::indicator:exclusive:unchecked:selected {
	image: url(:/Resources/Images/radiobutton_off.png);
}

QMenu::indicator:exclusive:checked {
	image: url(:/Resources/Images/radiobutton_on.png);
}

QMenu::indicator:exclusive:checked:selected {
	image: url(:/Resources/Images/radiobutton_on.png);
}*/

QAction:disabled {
	color: rgb(100, 100, 100);
}

/* QToolBar */
QToolBar {
	border: 0px solid rgb(100, 100, 100);
	border-width: 0, 1px, 0, 0;
}

QToolBar::separator {
	width: 1px;
	background-color: rgb(100, 100, 100);
	margin: 5px, 5px, 20px, 20px;
}

QToolBar QLabel {
	font-size: 11pt;
}

/* Button  Component*/
QMainWindow QPushButton {
    background: rgb(60, 60, 60);
	border: 1px solid rgb(200, 200, 200);
	border-radius: 3px;
	color: rgb(200, 200, 200);
	padding: 0px, 0px, 20px, 20px;
	min-width: 50px;
}

QMainWindow QPushButton:hover {
	background: rgb(80, 80, 80);
}

QMainWindow QPushButton:pressed {
	background: rgb(200, 200, 200);
	color: rgb(20, 20, 20);
}

QMainWindow  QPushButton.component {
	font-size: 11pt;
	min-width: 120px;
	padding: 5px, 5px;
}

/* Scrollbar */
Q/*ScrollBar {
    background-color: rgb(50, 50, 50);
	border: 1px solid rgb(100, 100, 100);
	border-radius: 4px;
}

QScrollBar:horizontal {
    height: 15px;
}

QScrollBar:vertical {
    width: 15px;
}

QScrollBar::add-page, QScrollBar::sub-page {
    background: none;
}

QScrollBar::add-line {
	 border: none;
	 background: none;
	width: 0px;
	height: 0px;
}

QScrollBar::sub-line {
	 border: none;
	 background: none;
	width: 0px;
}

QScrollBar::handle {
    background: rgb(60, 60, 60);
    min-width: 10px;
	border: 1px solid rgb(200, 200, 200);
	border-radius: 3px;
}

QScrollBar::handle:hover {
    background: rgb(80, 80, 80);
}*/

QScrollBar::handle:pressed {
    background: rgb(200, 200, 200);
}

/* QSlider */
QSlider:horizontal {
	min-height: 20px;
}

QSlider::groove {
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(100, 100, 100);
	height: 10px;
	border-radius: 3px;
}

QSlider::sub-page {
	background-color: rgb(200, 200, 200);
	border-radius: 3px;
}

QSlider::handle {
	background-color: rgb(50, 50, 50);
	border: 1px solid rgb(200, 200, 200);
	border-radius: 3px;
}

QSlider::handle:horizontal {
	width: 13px;
	margin-top: -5px;
	margin-bottom: -5px;
}

QSlider::handle:hover {
	background-color: rgb(80, 80, 80);
}

QSlider::handle:pressed {
	background-color: rgb(200, 200, 200);
	border-color: rgb(35, 35, 35);
}

/* QRadioButton */
QRadioButton::indicator::checked {
	color: rgb(200, 200, 200);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TimeRuler" name="timeRuler" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WaveFormView" name="waveFormView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="WaveFormScrollBar" name="waveFormScrollBar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="buttonLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="NodalView" name="nodalView">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="renderHints">
         <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::RubberBandDrag</enum>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::AnchorUnderMouse</enum>
        </property>
        <property name="resizeAnchor">
         <enum>QGraphicsView::AnchorViewCenter</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::MinimalViewportUpdate</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1054</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuComponents">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Components</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>Audio</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuComponents"/>
   <addaction name="menuAudio"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NodalView</class>
   <extends>QGraphicsView</extends>
   <header>NodalView.h</header>
  </customwidget>
  <customwidget>
   <class>WaveFormView</class>
   <extends>QWidget</extends>
   <header>WaveFormView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimeRuler</class>
   <extends>QWidget</extends>
   <header>TimeRuler.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WaveFormScrollBar</class>
   <extends>QWidget</extends>
   <header>WaveFormScrollBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
